k6/health.js

```diff
@@ -0,0 +1,10 @@
+import http from 'k6/http';
```

Comment:
Codacy has a fix for the issue: [Replace `'k6/http'` with `"k6/http"`](https://app.codacy.com/gh/dasch-swiss/dsp-api/pullRequest?prid=10006168)
```suggestion
import http from "k6/http";
```